### DISCUSSION TITLE
Cursor color & shape in the input (statusbar)

### DISCUSSION THREAD

**Discussion by macownersclub** [qutebrowser/qutebrowser#6534] [Q&A] [answered]
Hi everyone,

say: Is there a way to customize the shape/color of the cursor.
In my case it is almost not visible.

Now, I searched for a loong time and didn't find an answer. Please help. This drives me to the brink of madness.

Salut

**Comment by The-Compiler** [comment, marked as answer]
It's not customizable - you might be seeing this issue: #3981.

**Reply by macownersclub** [reply]
Wow... That was fast. thanx. I ll try and change the statusbar bg, then ;)
